docs/install.md

```diff
@@ -133,11 +133,21 @@ To **generate a new Guld identity** and PGP key:
 
 To **import an existing Guld identity** and PGP key:
 
-> **TODO** it looks like the UI for this one is broken right now. Choosing import doesn't change the UI at all.
-
 1. Select the "import" radio button.
 
-1. ...
+1. Export the private key you used to register your Guld Identity from your preferred PGP key manager
+
+1. Copy and paste the exported private key into the text area labeled ASCII Armored Private Key
+
+1. Enter your existing Guld username
+
+1. Enter a strong passphrase for the private PGP key, and confirm it.
+
+    Keep your passphrase stored securely! There is no way to recover a lost passphrase.
+    
+1. Click the import button to import your private key and Guld Identity
+
+   **Note** If you get an error when trying to use your existing Guld username checkmark the Expert Mode (DANGER!) option and try the import again
```

Comment:
"If you get an error when trying to use your existing Guld username, check the "Expert Mode" box and try the import again." (fix the wording, and add period.)

Reply:
better phrasing, thank  you!  :blush: